### DISCUSSION TITLE
Sidebar for images

### DISCUSSION THREAD

**Discussion by MTRNord** [dimsemenov/PhotoSwipe#1872] [Q&A] [answered]
Hi :) I am currently looking around, and I am wondering if I could use this to also display image specific comment sections next to the image if you open the image in the lightbox. So something like twitter for example on their webpage has. Didn't find any specific mention on it, but I probably missed a way to do this, so I just want to make sure this is possible before I start with this :)

**Comment by acwolff** [comment, marked as answer]
See thse examples: https://andrewolff.jalbum.net/ExtendedCommentSidePanel_PS/

**Reply by MTRNord** [reply]
Thanks it seems that is then done using https://github.com/dimsemenov/photoswipe-dynamic-caption-plugin after looking into the code of it :) Helped a lot 👍
